.github/actions/run-embedding-sdk-storybook/action.yml

```diff
@@ -0,0 +1,33 @@
+name: Run Embedding SDK Storybook
+runs:
+  using: "composite"
+  steps:
+    - name: Prepare frontend environment
+      uses: ./.github/actions/prepare-frontend
+
+    - name: Prepare back-end environment
+      uses: ./.github/actions/prepare-backend
+      with:
+        m2-cache-key: "cljs"
```

Comment:
What was the issue without prepare-frontend and prepare-backend in here?

Reply:
I understand that part, but we already prepare both frontend and backend in this job.